hwdb.d/70-mouse.hwdb

```diff
@@ -190,6 +190,17 @@ mouse:usb:v413cp301a:name:PixArt Dell MS116 USB Optical Mouse:*
 mouse:usb:v0461p4d46:name:USB Optical Mouse:*
  MOUSE_DPI=1000@125
 
+##########################################
+# Elecom
+#########################################
+
+# Elecom HUGE TrackBall (M-HT1DR)
+mouse:usb:v056ep010d:name:ELECOM TrackBall Mouse HUGE TrackBall:*
+ ID_INPUT_TRACKBALL=1
+ MOUSE_DPI=500@125 *1000@125 1500@125
```

Comment:
I don't know the format for this, @whot would be a better person for checking this.

Reply:
format is correct, thanks (fwiw hwdb parser too to catch errors for invalid formats)